store/store.go

```diff
@@ -194,6 +196,15 @@ func (s *Store) Commit() (root []byte, err lib.ErrorI) {
 	if e := s.writer.Flush(); e != nil {
 		return nil, ErrCommitDB(e)
 	}
+	// Trigger eviction of LSS deleted keys
+	// check if the current version is a multiple of the cleanup block interval
+	if s.Version()%s.config.StoreConfig.CleanupBlockInterval == 0 {
+		go func() {
+			if err := s.Evict(); err != nil {
```

Comment:
Is this function triggered anywhere else besides startup?

Reply:
no, is checked after every commit and runs every X configured blocks (default 200)